doc/user/content/sql/create-source/postgres.md

```diff
@@ -112,7 +118,10 @@ Materialize does not support changes to schemas for existing publications, and w
 
 ##### Supported types
 
-Sources can only be created from publications that use [data types](/sql/types/) supported by Materialize. Attempts to create sources from publications which contain tables with unsupported data types (like `enum` {{% gh 12689 %}}) will fail with an error.
+Replicating tables that contain [data types](/sql/types/) unsupported by Materialize is possible via the `TEXT COLUMNS` option. There are, however, known limitations for the following PostgreSQL types:
```

Comment:
I would slightly rephrase this because they are known limitations, but they're simply the way `text` columns behave:

> Replicating tables that contain [data types](/sql/types/) unsupported by Materialize is possible via the `TEXT COLUMNS` option. However, the data is simply treated as `text`, so ceases to offer the type's features. For example: